buildSrc/src/main/groovy/com/uber/okbuck/core/model/AndroidTarget.groovy

```diff
@@ -221,13 +221,19 @@ abstract class AndroidTarget extends JavaLibTarget {
         }.flatten() as Set<String>
     }
 
-    String getManifest() {
-        Set<String> manifests = [] as Set
-
-        baseVariant.sourceSets.each { SourceProvider provider ->
-            manifests.addAll(getAvailable(Collections.singletonList(provider.manifestFile)))
+    String getPackage(){
```

Comment:
This can be a potentially expensive operation to parse manifest for every `android_resource` in a target

We should add `@Memoized` to this as you mentioned this does not change